src/StudentPackage/TakingInputFromUser.java

```diff
@@ -0,0 +1,111 @@
+package StudentPackage;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.Comparator;
+public class TakingInputFromUser {
+
+    public static void TakingInput() throws IOException {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the no. of Student to enter the details");
+        int noOfStudents = sc.nextInt();
+        int rollNo = 0, ageOfStudent = 0;
+        String addressOfStudent = "", nameOfStudent = "";
+        Queue<String> setOfCourses = new LinkedList<>();
+
+        Student[] student = new Student[noOfStudents];
+
+        for (int i = 0; i < noOfStudents; i++) {
+
+            student[i] = new Student();
+            System.out.println("Enter the details of " + i + 1 + " Student");
+            System.out.println("Enter the roll no.");
+            rollNo = sc.nextInt();
+            System.out.println("Enter the age");
+            ageOfStudent = sc.nextInt();
+            System.out.println("Enter the name of student");
+            nameOfStudent = sc.next();
+            System.out.println("Enter the address of student");
+            addressOfStudent = sc.next();
+
+            System.out.println("enter the courses name whom you want to take , enter done when its over");
+
+            Set<String> course = new HashSet<String>();
+
+            while (true) {
+                String inputFromUserAboutTheCourses = sc.next();
+                if (((inputFromUserAboutTheCourses).toLowerCase()).equals("done"))
```

Comment:
Better to use a single character like (Y/N) instead of the word "done" 
You are building a user facing application, whenever you are expecting a user to type something in for your validation always keep it simple. Having type in a word can be more error prone than having to type in a letter